lib/mdx/plugins/url.ts

```diff
@@ -29,7 +29,14 @@ const rewriteLinkHref = (props: Props, match: UrlMatch): void => {
 	const { url, node, propertyName } = match;
 	if (node.tagName !== "a") return;
 	if (propertyName !== "href") return;
-	if (url.startsWith("http")) return;
+	// Open external links in new tab
+	if (url.startsWith("http")) {
+		node.properties = node.properties ?? {};
+		node.properties["href"] = url;
+		node.properties["target"] = "_blank";
+		node.properties["rel"] = "noreferrer";
+		return;
+	}
```

Comment:
What do you mean by https://github.com/thien-do/memos.pub/pull/39#issuecomment-1126793648 ?

I think this applies for absolute links already, no?

Reply:
Oh yeah, sorry, I meant `relative` link

Reply:
yeah relative links are a bit more complicated because we need to rewrite them to match the behaviour of github. Anyway, that was already done so I think we're good. Relative links don't really need to be opened in new tab like absolute ones.

Reply:
Cool